config/config.go

```diff
@@ -958,7 +958,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		TimeoutPrevoteDelta:         500 * time.Millisecond,
 		TimeoutPrecommit:            1000 * time.Millisecond,
 		TimeoutPrecommitDelta:       500 * time.Millisecond,
-		TimeoutCommit:               1000 * time.Millisecond,
+		TargetRoundDuration:         3500 * time.Millisecond,
 		SkipTimeoutCommit:           false,
```

Comment:
[for reviewers] noting `SkipTimeoutCommit=false` is already the case. Ref: https://github.com/tendermint/tendermint/issues/5911#issuecomment-967615517

Reply:
I think we should force this to be `false` (in app) else it will mess with this logic